src/structures/MessageSelectMenu.js

```diff
@@ -210,3 +210,8 @@ class MessageSelectMenu extends BaseMessageComponent {
 }
 
 module.exports = MessageSelectMenu;
+
+/**
+ * @external APISelectMenuButtonComponent
```

Comment:
Should be either Select Menu or Button, not both xD
```suggestion
 * @external APIMessageSelectMenu
```